Dockerfile.temporal

```diff
@@ -0,0 +1,23 @@
+FROM temporalio/auto-setup:1.20
+
+# Set up environment variables for PostgreSQL and Elasticsearch
+ENV DB=postgresql \
+    DB_PORT=5432 \
+    POSTGRES_USER=temporal \
+    POSTGRES_PWD=temporal \
+    POSTGRES_SEEDS=postgresql \
+    ENABLE_ES=true \
+    ES_SEEDS=elasticsearch \
+    ES_VERSION=v7
```

Comment:
`docker-compose.yml` using OpenSearch 2.5.0  while Temporal is configured for Elasticsearch v7. 
Let's update this to use a more recent version and ensure compatibility
```
ES_VERSION=v2
```